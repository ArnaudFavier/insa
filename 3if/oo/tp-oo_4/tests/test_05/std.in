R rectangle1 -8 4 4 -4
R rectangle2 0 0 1 1
LIST
MOVE rectangle2 -1 0
LIST
HIT rectangle1 -8 4
HIT rectangle1 4 -8
HIT rectangle1 1 0
HIT rectangle1 -8 5
HIT rectangle1 4 -5
HIT rectangle2 0 0
EXIT
