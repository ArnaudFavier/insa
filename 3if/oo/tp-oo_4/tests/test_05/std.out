OK
OK
rectangle1 R [(-8;4) (4;-4)]
rectangle2 R [(0;0) (1;1)]
OK
rectangle1 R [(-8;4) (4;-4)]
rectangle2 R [(-1;0) (0;1)]
YES
NO
YES
NO
NO
YES
